{"uuid": "b3c336d6-e689-42af-a653-63b92ec073f7", "children": ["df4b01e2-29b9-4986-9784-30e85fefbcbc"], "befores": [{"name": "browser_page", "status": "passed", "start": 1755005857599, "stop": 1755005859203}], "afters": [{"name": "browser_page::0", "status": "broken", "statusDetails": {"message": "Exception: BrowserContext.close: Connection closed while reading from the driver\n", "trace": "  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\alien\\Desktop\\ui_automationexercise\\fixtures\\browsers.py\", line 14, in browser_page\n    context.close()\n    ~~~~~~~~~~~~~^^\n  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 13482, in close\n    return mapping.from_maybe_impl(self._sync(self._impl_obj.close(reason=reason)))\n                                   ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ~~~~~~~~~~~^^\n  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_browser_context.py\", line 594, in close\n    await self.request.dispose(reason=reason)\n  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_fetch.py\", line 114, in dispose\n    await self._channel.send(\"dispose\", None, {\"reason\": reason})\n  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 69, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n    )\n    ^\n  File \"C:\\Users\\alien\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 558, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "start": 1755005864950, "stop": 1755005864955}], "start": 1755005857599, "stop": 1755005864955}